### PR TITLE
README: A few wording and layout edits

### PR DESCRIPTION
**Pull request by dbkinder**
Update the README with some spelling and readability changes, including an intra-page link (instead of describing where to jump to), and an update to the URL for commit message guidelines

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -79,7 +81,7 @@ While not recommended, you can also use Yocto Project bitbake directly.
 reproducible and debuggable by other developers using a different
 distribution.)
 
-Here are the basic steps, preparation::
+Here are the basic preperation steps::
```
Comment (gvancuts) [inline README.rst]: s/preperation/preparation ;)

File: README.rst [hunk 2]
```diff
@@ -2,17 +2,16 @@ intel-iot-refkit
 ################
 
 This README file contains information on setting up, building, and booting
-Iot Reference OS Kit for Intel(r) Architecture project.
+IoT Reference OS Kit for Intel(r) Architecture project.
```
Comment (gvancuts) [inline README.rst]: Should we add a "the" prior to "IoT Reference OS..." so the sentence would read: "...building, and booting the IoT Reference OS Kit for Intel(r)..."

Is it appropriate to have a comma followed by "and" ("...building, and booting...")

File: README.rst [hunk 3]
```diff
@@ -109,7 +111,7 @@ https://git-scm.com/docs/git-submodule
 Installing the Images
 =====================
 
-See detailed instructions in doc/howtos/image-install.rst.
+See detailed instructions in ``doc/howtos/image-install.rst``.
```
Comment (gvancuts) [inline README.rst]: Wouldn't it be better to introduce a cross-reference between the two docs? I.e.:
- Add `.. _image-install:` at the top of `doc/howtos/image-install.rst`
- Replace this string with `image-install`_

File: README.rst [hunk 4]
```diff
@@ -2,17 +2,16 @@ intel-iot-refkit
...
 a packaged and controlled environment for building an
-image, eliminating development environment issues that
+image and can eliminate development environment issues that
 can occur with differing Linux OS distros and versions, different host
-compilers, and such. (There are instructions later below for building
-without Docker.)
+compilers, and such. (We also describe `building without docker`_ in the instructions below.)
 
 Instructions for installing Docker for common Linux distros on your
 development machine (including Fedora and Ubuntu) are available at:
```
Comment (gvancuts) [inline README.rst]: Do we support non-Linux development systems (e.g. MacOS/Darwin or Windows)? If not, I would suggest making this clear here.

**Comment by gvancuts** [comment]
@dbkinder could you enhance the commit message too?

**Comment by mythi** [comment]
On Thu, Mar 9, 2017 at 10:21 AM, Geoffroy Van Cutsem wrote:

> @dbkinder <https://github.com/dbkinder> could you enhance the commit
> message too?
>

I'd also prefer adding 'README: ' prefix on the subject line.

**Comment by mythi** [comment]
On Thu, Mar 9, 2017 at 10:26 AM, Ylinen, Mikko wrote:

>
>
> On Thu, Mar 9, 2017 at 10:21 AM, Geoffroy Van Cutsem wrote:
>
>> @dbkinder <https://github.com/dbkinder> could you enhance the commit
>> message too?
>>
>
> I'd also prefer adding 'README: ' prefix on the subject line.
>

And speaking of the commit messages: something that's been on my TODO
is to fix the link about 'Patch Submission Guidelines' in the README. I had
put
a wrong link there as I originally intended to talk about how the commit
messages
should look like:
http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Headers_and_Commit_Messages

Could you take that fix in this PR too?

File: README.rst (hunk 3 above)
Comment (dbkinder) [inline README.rst]: Alas, ReST does not support cross-file references (that's a Sphinx thing using :ref:`label name`).  ReST directly supports intra-page links and hyperlinks (http protocol), but while we're delivering the documentation on github like this, we can't reference the other document files.  I think this is a temporary situation until we have this project's documentation as part of the overall YP document set.

Comment (gvancuts) [inline README.rst]: Thanks for the explanation @dbkinder . 👍 on your original modification then.

File: README.rst (hunk 2 above)
Comment (dbkinder) [inline README.rst]: Yes the "the" should be there. The comma before the and is a standard "oxford comma" and helps clarify the list of items.

File: README.rst (hunk 4 above)
Comment (dbkinder) [inline README.rst]: Mikko?  I suspect we only support what YP supports (namely Linux).

Comment (mythi) [inline README.rst]: Only docker on linux has been tested

**Comment by mythi** [comment]
@dbkinder can you squash the commits in one please

**Pull request closed without merging**